src/pytest_plugins/filler/filler.py

```diff
@@ -557,6 +569,11 @@ def pytest_html_report_title(report):
     report.title = "Fill Test Report"
 
 
+@pytest.fixture
```

Comment:
Can be moved to `src/pytest_plugins/shared/execute_fill.py` in order for this all to work with execute too.